src/fluree/crypto/jws.cljc

```diff
@@ -0,0 +1,96 @@
+(ns fluree.crypto.jws
+  "An implementation of https://datatracker.ietf.org/doc/html/rfc7515"
+  (:require [alphabase.core :as alphabase]
+            [clojure.string :as str]
+            [fluree.crypto.secp256k1 :as secp256k1]))
+
+(def JOSE-header
+  "The JOSE header for a secp256k1 signing key. https://github.com/decentralized-identity/EcdsaSecp256k1RecoverySignature2020"
+  "{\"alg\":\"ES256K-R\",\"b64\":false,\"crit\":[\"b64\"]}")
+
+(defn b64
+  "Convert to base64url and remove the trailing padding (=)."
+  [s]
+  (str/replace (alphabase/base-to-base s :string :base64url) "=" ""))
```

Comment:
minor: i think this would be more readable with the threading macro starting with `(-> s ...`